momento-sdk/src/test/java/momento/sdk/auth/StringCredentialProviderTest.java

```diff
@@ -8,13 +8,33 @@
 
 class StringCredentialProviderTest {
 
-  private static final String CONTROL_ENDPOINT = "control.example.com";
-  private static final String CACHE_ENDPOINT = "cache.example.com";
+  private static final String CONTROL_ENDPOINT_LEGACY = "control.example.com";
+  private static final String CACHE_ENDPOINT_LEGACY = "cache.example.com";
+  private static final String CONTROL_ENDPOINT_V1 = "control.test.momentohq.com";
+  private static final String CACHE_ENDPOINT_V1 = "cache.test.momentohq.com";
 
-  private static final String VALID_AUTH_TOKEN =
+  private static final String VALID_LEGACY_AUTH_TOKEN =
```

Comment:
We had a comment in the bug bash about having auth tokens in the tests. Would you mind adding a comment stating that these are fake?

Reply:
Sure, added in 7e0e8df